src/renderer/components/ui/Tooltip/Tooltip.tsx

```diff
@@ -131,7 +131,7 @@ export const Tooltip: React.FC<TooltipProps> = ({
   const classes = usestyles({})
   const [arrowRef, setArrowRef] = useState(null)
   return (
-    <span onClick={onClick}>
+    <span onClick={_e => onClick}>
```

Comment:
This look wrong. e is not passed to onClick

Reply:
Im getting `Type 'MouseEvent<HTMLSpanElement, MouseEvent>' is missing the following properties from type 'MouseEvent': offsetX, offsetY, x, y, and 13 more.ts(2345)` when I try to pass it

Reply:
hmm, the way you've done it, you never actually call `onClick`. Perhaps fix in TooltipPros:
```
(e: MouseEvent<HTMLSpanElement, MouseEvent>) => void
```

or maybe you dont need this event in the handler?